modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -987,6 +991,11 @@ private void _assertObjectDefinitions(
 		Assert.assertEquals(objectDefinition2.isSystem(), false);
 		Assert.assertEquals(
 			objectDefinition2.getStatus(), WorkflowConstants.STATUS_APPROVED);
+		Assert.assertEquals(
+			objectDefinition2.isAccountEntryRestricted(), false);
```

Comment:
We should remove assertEquals in this clause and use assertTrue or assertFalse instead